nmrfx-analyst-gui/src/main/java/org/nmrfx/processor/gui/PolyChart.java

```diff
@@ -166,9 +161,9 @@ protected PolyChart(FXMLController controller, String name, Pane plotContent, Ca
         this.canvas = canvas;
         this.peakCanvas = peakCanvas;
         this.annoCanvas = annoCanvas;
-        plotBackground = new Group();
         this.plotContent = plotContent;
         drawSpectrum = new DrawSpectrum(axes, canvas);
+        drawSpectrum.setCancelButton(controller.getCancelButton());
```

Comment:
Got NPE in drawSpectrum.setCancelButton because at this point the controller has not yet created the cancel button.
See lines 842 (polyChart creation) and 844 (cancelbutton creation) in FXMLController